muxpi/nuc/ethup.sh

```diff
@@ -0,0 +1,18 @@
+ #!/bin/bash
+ DUT_IP="$1"
+ n=1 
+ max=5
+ delay=50
+ while true; do
+    ping -c 1 -W 1 $DUT_IP && break || {
```

Comment:
Is timeout value of 1 ms enough for the ping reply?

Reply:
Sorry, ignore my comment, the timeout is in seconds.

Reply:
The delay is 50s, 5 attempts to ping.

Reply:
(ping -c 20 -W 50000 172.31.173.180 && echo "success") || echo "failed"